src/mid.cpp

```diff
@@ -438,6 +449,8 @@ void CmidPlayer::midi_fm_playnote(int voice, int note, int volume)
     int oct=note/12;
 	int c;
 
+    midiprintf ("playnote(voice %d, note %d, volume %d)\n", voice, note, volume);
```

Comment:
Can these (and the other) debug statements be removed?  It looks like they are left over from your testing.